api/app/src/external/commonwell/organization.ts

```diff
@@ -0,0 +1,83 @@
+import { Organization as CWOrganization } from "@metriport/commonwell-sdk";
+
+import { Config, getEnvVarOrFail } from "../../shared/config";
+import { commonWellMember, queryMeta, CW_ID_PREFIX } from "../../shared/commonwell";
+import BadRequestError from "../../errors/bad-request";
+import { createOrgId } from "../../shared/oid";
+import { Organization } from "../../routes/medical/models/organization";
```

Comment:
I think we should leave the types representing boundaries at the "outer layers" of the application:
- `src/routes/medical/models/.*` represent the API schema, or "API input" - we keep it on the "API layer";
- this file is part of another boundary, "integration layer";
- the connecting part between them should be our "application layer" - code that coordinates what calls what (commands, "err... routes")
- to connect the API layer to the integration layer, we should go through the application layer
- the common language on the application layer is the domain
  - currently, we don't have real domain classes, we're using DB models as domain
  - but that's fine for now, and better to use those on the application layer than boundary ones

So, ideally we would convert API schemas into domain/model types, and those would be the input of this file (which represents a boundary with Commonwell). Here we would convert from the domain/model to the types Commonwell understands.

Reply:
Im assuming this is resolved with this  https://github.com/metriport/metriport/pull/131